_integration-schemas/selligent/internal_datasource.md

```diff
@@ -0,0 +1,54 @@
+---
+tap: "selligent"
+version: 1.0
+key: ""
+
+name: "internal_datasource"
+doc-link: 
+singer-schema: https://github.com/singer-io/tap-selligent/blob/master/tap_selligent/schemas.py
+description: "This table contains internal data source data."
+
+replication-method: "Full Table"
+
+replication-key:
+  name: ""
+
+api-method:
+  name: ""
+  doc-link: ""
+
+attributes:
+  - name: "id"
+    type: "integer"
+    primary-key: true
+    description: "The asset_id of this mailing."
+    foreign-key-id: "internal-id"
```

Comment:
Just like with Klaviyo, let's comment this out for now since it doesn't point to anything.

Ex:

```suggestion
    # foreign-key-id: "internal-id"
```